synapse/lib/certdir.py

```diff
@@ -944,6 +944,17 @@ def getUserKeyPath(self, name):
             if os.path.isfile(path):
                 return path
 
+    def getUserCsrPath(self, name):
+        for cdir in self.certdirs:
+            path = s_common.genpath(cdir, 'users', '%s.csr' % name)
```

Comment:
Nit: I'd rather see an f-string here

Reply:
Agreed; but there is value in keeping the certdir code with a consistent voice until we rewrite it ;)

Reply:
It's alright, lets adjust and stop digging :+1: